docs/webapp/datasets/webapp_dataset_viewing.md

```diff
@@ -83,7 +83,7 @@ Access these actions with the context menu by right-clicking a version on the da
 | Action | Description | 
 |-----|----|
 |Add Tag |User-defined labels added to versions for grouping and organization. |
-|Archive| To more easily work with active dataset versions, archive a version. [See Archiving](../webapp_archiving.md).| 
+|Archive| To more easily work with active dataset versions, archive a version. Archived datasets do not appear in the active (main) dataset versions table.| 
```

Comment:
@pollfly "To more easily...runs table" -> "Move dataset versions to the dataset's archive" (same for pipelines etc.)